src/sentry/web/client_config.py

```diff
@@ -221,6 +221,8 @@ def enabled_features(self) -> Iterable[str]:
             "organizations:multi-region-selector", actor=self.user
         ):
             yield "organizations:multi-region-selector"
+        if features.has("organizations:react-concurrent-renderer", actor=self.user):
```

Comment:
I think you could just implement this using `options`

Reply:
i thought that is what i did here are they not the same thing